src/main/java/org/eclipse/yasson/internal/ClassParser.java

```diff
@@ -263,15 +275,13 @@ private void checkPropertyNameClash(List<PropertyModel> collectedProperties, Cla
         for (PropertyModel collectedPropertyModel : collectedProperties) {
             for (PropertyModel checkedPropertyModel : checkedProperties) {
 
-                if ((
-                        checkedPropertyModel.getReadName().equals(collectedPropertyModel.getReadName())
-                                && checkedPropertyModel.isReadable() && collectedPropertyModel.isReadable())
+                if ((checkedPropertyModel.getReadName().equals(collectedPropertyModel.getReadName())
+                        && checkedPropertyModel.isReadable() && collectedPropertyModel.isReadable())
                         || (checkedPropertyModel.getWriteName().equals(collectedPropertyModel.getWriteName()))
-                        && checkedPropertyModel.isWritable() && collectedPropertyModel.isWritable()) {
-                    throw new JsonbException(Messages.getMessage(MessageKeys.PROPERTY_NAME_CLASH,
-                                                                 checkedPropertyModel.getPropertyName(),
-                                                                 collectedPropertyModel.getPropertyName(),
-                                                                 cls.getName()));
+                                && checkedPropertyModel.isWritable() && collectedPropertyModel.isWritable()) {
```

Comment:
I might recommend to change this a bit. Would you mind to make it like this? It improves readability and it is also not incorrectly formatted by IDE.
```
    private void checkPropertyNameClash(List<PropertyModel> collectedProperties, Class cls) {
        ....
                if (isReadable(checkedPropertyModel, collectedPropertyModel)
                        || isWritable(checkedPropertyModel, collectedPropertyModel)) {
        ....
    }

    private boolean isReadable(PropertyModel checkedPropertyModel, PropertyModel collectedPropertyModel) {
        return checkedPropertyModel.getReadName().equals(collectedPropertyModel.getReadName())
                && checkedPropertyModel.isReadable()
                && collectedPropertyModel.isReadable();
    }

    private boolean isWritable(PropertyModel checkedPropertyModel, PropertyModel collectedPropertyModel) {
        return checkedPropertyModel.getWriteName().equals(collectedPropertyModel.getWriteName())
                && checkedPropertyModel.isWritable()
                && collectedPropertyModel.isWritable();
    }
```